ecmd-core/ext/fapi2/capi/plat_hw_access.C

```diff
@@ -1739,53 +1579,64 @@ const fapi2::TargetType TARGET_TYPE_SCOM_TARGET_14_11_NO_Z = fapi2::TARGET_TYPE_
                               fapi2::variable_buffer& o_data,
                               const fapi2::RingMode i_ringMode)
     {
-        return getRingTemplate<>(i_target, i_address, o_data, i_ringMode);
+        return fapi2plat::getRing<>(i_target, i_address, o_data, i_ringMode);
     }
 #endif
 
     fapi2::ReturnCode putRing(const fapi2::Target<fapi2plat::TARGET_TYPE_SCAN_TARGET, fapi2::plat_target_handle_t>& i_target,
                               const RingId_t i_ringID,
                               const fapi2::RingMode i_ringMode)
     {
-        return putRingTemplate<>(i_target, i_ringID, i_ringMode);
+        return fapi2plat::putRing<>(i_target, i_ringID, i_ringMode);
     }
 
     fapi2::ReturnCode putRing(const fapi2::Target<fapi2plat::TARGET_TYPE_SCAN_TARGET_14_5, fapi2::plat_target_handle_t>& i_target,
                               const RingId_t i_ringID,
                               const fapi2::RingMode i_ringMode)
     {
-        return putRingTemplate<>(i_target, i_ringID, i_ringMode);
+        return fapi2plat::putRing<>(i_target, i_ringID, i_ringMode);
     }
 
 #ifdef FAPI_2_Z
     fapi2::ReturnCode putRing(const fapi2::Target<fapi2plat::TARGET_TYPE_SCAN_TARGET_14_6, fapi2::plat_target_handle_t>& i_target,
                               const RingId_t i_ringID,
                               const fapi2::RingMode i_ringMode)
     {
-        return putRingTemplate<>(i_target, i_ringID, i_ringMode);
+        return fapi2plat::putRing<>(i_target, i_ringID, i_ringMode);
     }
 #endif
 
+    template< fapi2::TargetType K, typename V >
```

Comment:
This function looks out of place in the .C and it's already defined in the .H, is there a reason behind it?

Reply:
This is the deprecated putRing interface with RingID type. It is no longer available in the header, but we wanted to support procedures that had compiled against this interface.

Reply:
Ohh, i notice the subtle difference now -- RingID vs. RingId_t :mag: 
Never mind then :)